src/App.test.jsx

```diff
@@ -23,12 +21,46 @@ test('App', () => {
     ],
   }));
 
-  const { queryByText } = render((
-    <App />
-  ));
+  useDispatch.mockImplementation(() => dispatch);
+
+  function renderApp({ path }) {
+    return render(
+      <MemoryRouter initialEntries={[path]}>
+        <App />
+      </MemoryRouter>,
+    );
```

Comment:
```suggestion
    return render((
      <MemoryRouter initialEntries={[path]}>
        <App />
      </MemoryRouter>
    ));
```

괄호를 한번 더 감싸면 콤마를 제거할 수 있어요!

Reply:
좋은 피드백 감사합니다!
리팩토링 진행하겠습니다!